test/integration/user_logs_in_with_facebook_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+class UserLogsInWithTwitterTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  def setup
+    Capybara.app = FeedBee::Application
+    stub_omniauth
+  end
+
+  def stub_omniauth
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+      provider: 'facebook',
+      uid: '123',
+      info: {
+        name: "Ryan Asensio"
+      },
+      credentials: {
+        oauth_token: "pizza",
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.